docs/user/openstack/README.md

```diff
@@ -521,6 +520,10 @@ To define a MachineSet with multiple networks, the `primarySubnet` value in the
 
 #### Using a Server Group
 
+The `serverGroupID` property of the MachineSet will be used to create new
+machines in that OpenStack Server group. The server group must exist in
+OpenStack before the new MachineSet is applied.
+
```

Comment:
```suggestion
The `serverGroupID` property of the [machine set | `MachineSet` resource] is used to create machines in that OpenStack server group. The server group must exist in OpenStack before you can apply the new [machine set | `MachineSet` resource].
```

Since I'm looking. :)